### PR TITLE
WELD-2316 port useful methods from SeContainerInitializer.

### PR DESCRIPTION
**Pull request by manovotn**

### CODE REVIEW

**Comment by manovotn** [comment]
Will also add methods such as `enableInterceptors`

File: environments/se/core/src/main/java/org/jboss/weld/environment/se/Weld.java [hunk 1]
```diff
@@ -345,6 +345,32 @@ public Weld addPackage(boolean scanRecursively, Class<?> packageClass) {
     }
 
     /**
+     * Provided Packages be scanned and found classes will be added to the set of bean classes for the synthetic bean archive.
```
Comment (mkouba) [inline environments/se/core/src/main/java/org/jboss/weld/environment/se/Weld.java]: "...will be scanned..."?

Comment (manovotn) [inline environments/se/core/src/main/java/org/jboss/weld/environment/se/Weld.java]: Affirmative

**Comment by mkouba** [comment]
Merged, thanks!

**Pull request closed without merging**